doitlive.py

```diff
@@ -22,9 +22,13 @@
 from code import InteractiveConsole
 from tempfile import NamedTemporaryFile
 from collections import OrderedDict
+from termutils import raw_mode
 
 import click
-from click import echo, style, secho, getchar
+if int(click.__version__[0]) >= 5:
+    click.disable_unicode_literals_warning = True  # this fixes #12
```

Comment:
I completely agree about this, although I'd rather handle this in another issue/pull request. This is supposed to be a quick fix in the meantime. (As no bug regarding Unicode literals is known right now for doitlive, I don't think it's urgent)